spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -1105,6 +1105,9 @@ A schedule can be created through the RESTful API or the Spring Cloud Data Flow
 Spring Cloud Data Flow will schedule the execution of its tasks via a scheduling agent that is available on the cloud platform.
 When using the Cloud Foundry platform Spring Cloud Data Flow will use the https://www.cloudfoundry.org/the-foundry/scheduler/[PCF Scheduler].
 When using Kubernetes, a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] will be used.
+The way that Spring Cloud Data Flow does this is by deploying a SchedulerTaskLauncher application to the platform and then
+the scheduling agent is notified to schedule the SchedulerTaskLauncher.  When the scheduling agent launches the
```

Comment:
"scheduling agent" <- sounds generic; maybe add in parens: "(e.g., CronJob in Kubernetes)"

Reply:
For "SchedulerTaskLauncher", do we want to link to the GH module?

Reply:
Updated.  Thank you!